python_modules/libraries/dagster-sigma/dagster_sigma/translator.py

```diff
@@ -53,16 +54,35 @@ class SigmaDataset:
     inputs: AbstractSet[str]
 
 
+@whitelist_for_serdes
```

Comment:
does this need to be its own serdes-able type? just seems mildly sketchy given that it's really just wrapping an arbitrary dictionary.

I suppose that for these particular types of serdes objects we don't need to worry about backcompat so it's not the end of the world or anything, just a bit odd